src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -752,6 +752,18 @@ public RemoteOutputsStrategyConverter() {
               + " smart enough about the RAM/CPU usages, this flag will be removed.")
   public boolean throttleRemoteActionBuilding;
 
+  @Option(
+      name = "experimental_remote_output_service",
```

Comment:
Even though the Bazel Output Service protocol we checked in has a way where the server can tell the client what the absolute path of the mount is, this is optional. In the case of bb_clientd I'd prefer it if clients continue to pass in the path of the mount. Could we therefore add a flag for that as well? Thanks!

Reply:
Yes, that flag will be added in a follow-up PR.